src/scripts/public/index.js

```diff
@@ -1,3 +1,4 @@
+import '@bcgov/design-tokens/css-prefixed/variables.css';
```

Comment:
so this is cool that you are using the design tokens, now how do we tell our future selfs, that these tokens are in the styles, i almost feel like a header comment block above the declaration would be helpful, to indicate their use and what they look like ie `var(--bcds-surface-brand-gray-30)`

Reply:
for this matter, we might want to do this for all of our var(--wp-sample-color), to indicate what are wordpress out of the box, and what ones belong to the theme.  Setting a standard of sorts

Reply:
I have added a JSDoc to the top of this and the admin/index.js to explain the Alpha styles and prefixed variables relationship.